errata_tool/erratum.py

```diff
@@ -834,12 +863,13 @@ def _write(self):
 
         # XXX Delete all bugs is a special case
         last_bug = None
-        if len(self.errata_bugs) == 0 and len(self._original_bugs) > 0:
-            last_bug = self._original_bugs[0]
+        if len(self.errata_bugs) == 0 and len(self._original_errata_bugs) > 0:
```

Comment:
This logic probably needs to continue using `_original_bugs` because we'll probably need the same behavior wither we have the last Bugzilla bug or the last Jira issue.

Also we need to check wither all bugs and all issues were removed here, so perhaps set `allbugs` first and use that to check.

Reply:
Hey @ifireball Good news, I tested this case today with 1 bugzilla bug attached to an advisory (https://errata.devel.redhat.com/advisory/96435) and I ET didn't throw an error when idsfixed was an empty string and it did drop the bug. So it looks to me they fixed the issue. 

I have removed that logic and don't think we need to account for that now. Let me know if you want to verify it